Add new reference module for expiring RHEL support (#2617)
theforeman/foreman-documentation: guides/common/modules/proc_accessing-rhel-lifecycle-status.adoc

[id="Accessing_RHEL_Lifecycle_Status_{context}"]

> = Accessing RHEL Lifecycle Status

Reviewer: ```suggestion
= Displaying RHEL Lifecycle Status
```

> Use the following procedure to access the status of the end of support (EOS) for your {RHEL} clients.

Reviewer: ```suggestion
You can display the status of the end of support (EOS) for your {RHEL} hosts in the table on the hosts index page.
```

.Procedure
. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
. Click *Manage columns*.
. Select the *Content* column to expand it.
. Select *RHEL Lifecycle status*.
. Click *Save* to generate a new column that displays the {EL} lifecycle status.

You can also use the *Search* field to search hosts by `rhel_lifecycle_status`.
It can have the following values:

* `full_support`
* `maintenance_support`
* `approaching_end_of_maintenance`
* `extended_support`
* `approaching_end_of_support`

> * `support_ended`

Reviewer: I think the search could be split into an individual module.

You can also find the {EL} lifecycle status on the *Host status* card on the host details page.

> You will see an alert on the host page for each host that runs {EL} with an upcoming End of Support (EOS) event in a year as well as when support has ended.

Reviewer: ```suggestion
You will see an alert on the host index page for each host that runs {EL} with an upcoming End of Support (EOS) event in a year as well as when support has ended.
```
Is this the page you have in mind?

> You will see an alert on the host page for each host that runs {EL} with an upcoming End of Support (EOS) event in a year as well as when support has ended.

Reviewer: I think these two sentences could be in the concept module at the beginning.
